backend/src/api.py

```diff
@@ -0,0 +1,61 @@
+from datetime import datetime
```

Comment:
in current version this import is unused - can be useable when https://github.com/dgrafik/personal-budget/pull/2/files#r457743407 will be applied